src/animation/linear_animation_instance.cpp

```diff
@@ -112,5 +130,7 @@ void LinearAnimationInstance::time(float value)
 	m_TotalTime = value - start;
 	m_LastTotalTime = m_TotalTime - diff;
 
+	// leaving this RIGHT now. but is this required? it kinda messes up
+	// playing things backwards and seeking. what purpose does it solve?
 	m_Direction = 1;
```

Comment:
This was probably because we weren't dealing well with changing time when duration was backwards. Need to run some numbers to make sure total time and last total time look good without it (at first glance it seems like it should be ok). Most of that implicates the exit time condition for certain states in the state machine...